cms/jinja2/templates/pages/statistical_article_page--related_data.html

```diff
@@ -0,0 +1,52 @@
+{% extends "templates/base_page.html" %}
+
+{% from "components/document-list/_macro.njk" import onsDocumentList %}
+{% from "components/hero/_macro.njk" import onsHero %}
+{% from "components/pagination/_macro.njk" import onsPagination %}
+
+{% block header_area %}
+    {# fmt:off #}
+    {{
+        onsHero({
+            "topic": _("Related data"),
```

Comment:
could you extract the translatable strings outside of this. It looks like gettext doesn't pick on them for some reason. An example:

```jinja2
{%- set hero_topic = _("Related data") -%}
{%- set breadcrumbs_aria_label = _("Breadcrumbs") -%}
```